logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -70,15 +72,13 @@ sealed class Event(open val id: String, open val transient: Boolean) {
             val qualifiedFrom: UserId,
             override val transient: Boolean
         ) : Conversation(id, transient, conversationId) {
-            override fun toLogString(): String {
-                val properties = mapOf(
-                    typeKey to this::class.qualifiedName,
+
+            override fun toLogMap(): Map<String, Any?> = mapOf(
+                    typeKey to this::class.simpleName,
```

Comment:
Just a reminder that this will be minified on release builds.

Not great, but we can always map back from the mapping file.

Reply:
Change to something meaningful